src/utils/choice_random_items.js

```diff
@@ -0,0 +1,11 @@
+function randomChoiceItems(length) {
+    const result = [];
+    for (let i = 0; result.length < 2; i++) {
```

Comment:
I think lenght should be used here as well :)
btw, let's rename the variable length to maxNumberOfRandomItems